mods/ra/maps/exodus/exodus.lua

```diff
@@ -0,0 +1,398 @@
+	"apc", "e1", "e1", "e2", "e3", "e3", "e4"
+}
+
+SovietUnits2 =
+{
+	"4tnk", "4tnk", "4tnk", "4tnk", "3tnk", "3tnk", "3tnk", "3tnk", "v2rl",
+	"v2rl", "ftrk", "apc", "e1", "e1", "e2", "e3", "e3", "e4"
+}
+
+CurrentReinforcement1 = 0
+CurrentReinforcement2 = 0
+SpawnAlliedUnit = function(units)
+	Reinforcements.Reinforce(allies1, units, { Allies1EntryPoint.Location, Allies1MovePoint.Location })
+
+	if allies2 then
+		Reinforcements.Reinforce(allies2, units, { Allies2EntryPoint.Location, Allies2MovePoint.Location })
+	end
+
+	Utils.Do(humans, function(player)
+		if player and player.IsLocalPlayer then
+			Trigger.AfterDelay(DateTime.Seconds(2), function()
+				Media.PlaySpeechNotification(player, "AlliedReinforcementsNorth")
+			end)
+		end
+	end)
+
+	if CurrentReinforcement1 < #Reinforcements1 then
+		CurrentReinforcement1 = CurrentReinforcement1 + 1
+		Trigger.AfterDelay(ReinforcementsTicks1, function()
+			reinforcements1 = { Reinforcements1[CurrentReinforcement1] }
+			SpawnAlliedUnit(reinforcements1)
+		end)
+	end
+
+	if CurrentReinforcement2 < #Reinforcements2 then
+		CurrentReinforcement2 = CurrentReinforcement2 + 1
+		Trigger.AfterDelay(ReinforcementsTicks2, function()
+			reinforcements2 = { Reinforcements2[CurrentReinforcement2] }
+			SpawnAlliedUnit(reinforcements2)
+		end)
+	end
+end
+
+SovietGroupSize = 5
+SpawnSovietUnits = function()
+	spawnPoint = Utils.Random(SovietEntryPoints)
+	rallyPoint = Utils.Random(SovietRallyPoints)
+	route = { spawnPoint.Location, rallyPoint.Location }
+
+	if DateTime.GameTime >= SovietUnits2Ticks[Difficulty] then
+		units = SovietUnits2
+	else
+		units = SovietUnits1
+	end
+
+	unit = { Utils.Random(units) }
+	Reinforcements.Reinforce(soviets, unit, route)
+
+	delay = math.max(attackAtFrame - 5, minAttackAtFrame)
+
+	Trigger.AfterDelay(delay, SpawnSovietUnits)
+end
+
+SovietParadrop = 0
+SendSovietParadrop = function()
+	local sovietParadrops = SovietParadrops[Difficulty]
+
+	if (SovietParadrop > sovietParadrops) then
+		return
+	end
+
+	SovietParadrop = SovietParadrop + 1
+
+	Utils.Do(humans, function(player)
+		Media.PlaySpeechNotification(player, "SovietForcesApproaching")
+	end)
+
+	local x = Utils.RandomInteger(ParadropBoxTopLeft.Location.X, ParadropBoxBottomRight.Location.X)
+	local y = Utils.RandomInteger(ParadropBoxBottomRight.Location.Y, ParadropBoxTopLeft.Location.Y)
+
+	local randomParadropCell = CPos.New(x, y)
+	local lz = Map.CenterOfCell(randomParadropCell)
+
+	local powerproxy = Actor.Create("powerproxy.paratroopers", false, { Owner = soviets })
+	powerproxy.SendParatroopers(lz)
+	powerproxy.Destroy()
+
+	Trigger.AfterDelay(sovietParadropTicks, SendSovietParadrop)
+end
+
+IdleHunt = function(unit)
+	Trigger.OnIdle(unit, unit.Hunt)
+end
+
+ManageSovietUnits = function()
+	Utils.Do(SovietRallyPoints, function(rallyPoint)
+		local radius = WDist.FromCells(8)
+		local units = Map.ActorsInCircle(rallyPoint.CenterPosition, radius, function(actor)
+			return actor.Owner == soviets
```

Comment:
I suppose adding `and actor.HasProperty("Hunt")` here would fix the crash I saw.